paperless-ngx.json

```
@@ -0,0 +1,113 @@
+{
+    "Paperless-ngx": {
+        "container_links": {
+            "paperless": [
+                {
+                    "name": "paperless-to-broker",
+                    "source_container": "paperless-broker"
+                },
+                {
+                    "name": "paperless-to-paperless-db",
+                    "source_container": "paperless-db"
+                }
+            ]
+        },
+        "containers": {
+            "paperless": {
+                "image": "ghcr.io/paperless-ngx/paperless-ngx",
+                "launch_order": 3,
```

```suggestion
                "launch_order": 3,
"opts": [
                    [
                        "-e",
                        "PAPERLESS_DBHOST=paperless-db"
                    ],
                    [
                        "-e",
                        "PAPERLESS_REDIS=redis://paperless-broker:6379"
                    ]
                ],
```
Since these values don't really need to change in this Rockon context, I suggest these can be moved into the options sections and so won't require a user to enter these values verbatim. Changes to these would likely require changes to the overall json anyway, and if an advanced user so desires, they will likely create their own version of this.